bin/split-silo-database

```diff
@@ -0,0 +1,83 @@
+#!/usr/bin/env python
+
+from typing import List
+
+import click
+from django.apps import apps
+
+import docker
+from sentry.runner import configure
+from sentry.silo.base import SiloMode
+
+configure()
+
+
+def exec_run(container, command):
+    wrapped_command = f'sh -c "{" ".join(command)}"'
+    exit_code, output = container.exec_run(cmd=wrapped_command, stdout=True, stderr=True)
+    if exit_code:
+        click.echo("Container operation Failed!")
+        click.echo(f"Container operation failed with {output}")
+
+
+def split_database(tables: List[str], source: str, destination: str, reset: bool, verbose: bool):
+    click.echo(f">> Dumping tables from {source} database")
+    command = ["pg_dump", "-U", "postgres", "-d", source, "--clean"]
+    for table in tables:
+        command.extend(["-t", table])
+    command.extend([">", f"/tmp/{destination}-tables.sql"])
```

Comment:
This may be a nit, but I bet users would prefer greatly being able to provide a full path for this, or atleast a prefix.  But super minor.

Reply:
Also, if this file already exists, AND we've dropped the existing database, shouldn't we just use the existing file?  Thinking about failure cases, it would suck if we dump the database, drop it, but fail to rebuild from the dump file and then just be stuck.

Ideally, if the dump file remains, we'd be able to provide it as input on he retry, if we don't still have the destination db to redump from.

Reply:
Oh, well, I do see that reset is optional, so maybe that's totally fine.

Reply:
This file is inside the postgres container, not the host system. Do you think the path still needs to be configurable?